Update Broken links in KFServing documentation

tcnt opened the pull request:
Update links to SDK client example.
Update link to Knative monitoring. Replace it by a link to kfserving github, where the details can be found.
See issue https://github.com/kubeflow/website/issues/2329

Jeffwan commented:
/cc @yuzisun @animeshsingh

animeshsingh commented on content/en/docs/components/serving/kfserving.md:
```diff
@@ -74,7 +74,7 @@ Kubeflow 1.1 includes KFServing v0.3, where the focus has been on providing more
 ### Sample notebooks
 
-* [SDK client](https://github.com/kubeflow/kfserving/blob/master/docs/samples/client/kfserving_sdk_sample.ipynb)
+* [SDK client](https://github.com/kubeflow/kfserving/blob/master/docs/samples/client)
```
```suggestion
* [SDK client](https://github.com/kubeflow/kfserving/blob/master/docs/samples/client/kfserving_sdk_v1alpha2_sample.ipynb)
```

tcnt replied:
@animeshsingh I had it on purpose on the directory level, since there are now 2 notebooks.
By using the directory, the link does not need to change if the underlying notebook(s) change again.
I did the same in the readme on kfserving kubeflow/kfserving#1193 which was approved by @yuzisun

animeshsingh replied:
hear you - but the website docs point to an official release, and that section is title "notebooks" specifically. The 2nd notebook is still not official for overall Kubeflow

tcnt replied:
ok. Clear. But should I then not also change the same link in the SDK Section?

animeshsingh replied:
it's fine there. no strong opinion

animeshsingh commented:
/ok-to-test

8bitmp3 commented on content/en/docs/components/serving/kfserving.md:
```diff
@@ -23,7 +23,7 @@ Our strong community contributions help KFServing to grow. We have a Technical S
 
 KFServing works with Kubeflow 1.1. Kustomize installation files are [located in the manifests repo](https://github.com/kubeflow/manifests/tree/master/kfserving).
-See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex). For installation on major cloud providers with Kubeflow, please follow their installation docs. 
+See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex). For installation on major cloud providers with Kubeflow, please follow their installation docs.
```
- For a11y + rephrase
```suggestion
Check the examples running KFServing on Istio/Dex in the [`kubeflow/kfserving`](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex) repository. For installation on major cloud providers with Kubeflow, follow their installation docs.
```

Would it be possible to find links to AWS etc KFServing installation guides and include them here? This can help new users 💯, I think

tcnt replied:
The text above refers to follow the installation docs of the major cloud providers.
Examples should then be in the existing links like https://www.kubeflow.org/docs/aws/
I do not think we should additional links on the current page.

8bitmp3 commented on content/en/docs/components/serving/kfserving.md:
````diff
@@ -105,19 +105,15 @@ Once you meet the above prerequisites KFServing can be [installed standalone](ht
 
 ## Use SDK
 
 1. Install the SDK.
     ```
     pip install kfserving
     ```
````
```suggestion
1. Install the SDK with PiPy.

    ```shell
    pip install kfserving
    ```

```

8bitmp3 reviewed (changes requested):
PTAL @tcnt 👍 

Thank you for the help!

8bitmp3 commented:
/lgtm 

thanks @tcnt 

/assign @adrian555 @animeshsingh @cliveseldon @ellistarn @ryandawsonuk @Tomcli @yuzisun
(OWNERS-approvers)

yuzisun commented:
/lgtm

animeshsingh commented:
@tcnt looks like you need to rebase before we merge

tcnt commented:
> @tcnt looks like you need to rebase before we merge

@animeshsingh I actually had to solve a merge conflict, as 1 of the lines I updated was also updated in the pull request https://github.com/kubeflow/website/pull/2342 which was merged into the master yesterday.
As far as I can see it should be ok now

animeshsingh commented:
/lgtm
/approve

iancoffey commented:
Unfortunately, looks like this PRs updates still direct folks to broken 404ed monitoring resources - try the metrics link in the github markdown, for example: https://github.com/kubeflow/kfserving#setup-monitoring

I think this monitoring situation still need a solution, and hope we can discuss inhttps://github.com/kubeflow/kfserving/pull/1162 as this is still a dead-end afaict.

tcnt commented:
@yuzisun Can you please have a look at this? The changes made in the current pull request where made assuming https://github.com/kubeflow/kfserving/pull/1162 would merge soon as well. As we discussed in https://github.com/kubeflow/website/issues/2329 you proposed to make the changes to refer to [KFS monitoring section](https://github.com/kubeflow/kfserving#setup-monitoring)
